ui/src/tests/setup.ts

```diff
@@ -19,14 +19,23 @@ const windowsPrefix = process.platform === 'win32' ? '.exe' : '';
 const appDotGo = path.join(__dirname, '..', '..', '..', 'app.go');
 const testBuildPath = path.join(__dirname, 'build');
 
-export const newTest = async (): Promise<GotifyTest> => {
+export const newTest = async (plugins?: string[]): Promise<GotifyTest> => {
```

Comment:
Could we only pass the plugin dir here, and build the plugins in a separate method that is only called in the plugin tests?